Gruntfile.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+module.exports = function (grunt) {
```

Comment:
Perhaps adopting some of the generator conventions could be done in a piecemeal way with this project in subsequent PRs. I was thinking the same about some of the bigger projects, which will not work with the generator's default setup (namely because they assume ES6 everything as does the linter).